core/src/identity.rs

```diff
@@ -179,7 +179,7 @@ impl PublicKey {
 
     /// Encode the public key into a protobuf structure for storage or
     /// exchange with other nodes.
-    pub fn into_protobuf_encoding(self) -> Vec<u8> {
+    pub fn into_protobuf_encoding(&self) -> Vec<u8> {
```

Comment:
Per Rust-convention, this would now have to be renamed to `to_protobuf_encoding`.